### PR TITLE
Update to Zeebe 8.0.6

### PR DESCRIPTION
**Pull request by saig0**
- dump Zeebe from 8.0.5 to 8.0.6
- dump Zeebe's go client from 8.0.5 to 8.0.6

### CODE REVIEW

**Comment by npepinpe** [comment]
Do we still need to wait for all the releases before merging these PRs?

**Comment by saig0** [comment]
@npepinpe for me, it would be fine to merge the PR already. The artifacts are released. So, I see no blocker for the users.

**Comment by npepinpe** [comment]
Before I would wait because there was also some Operate/Tasklist versions to update, but I think this isn;'t the case anymore. So you're right, we can probably merge already.

**Review by npepinpe — APPROVED** [review]

**Pull request merged**
